llms/mlx_lm/tuner/utils.py

```diff
@@ -6,18 +6,60 @@
 
 
 def apply_lora_layers(model: nn.Module, adapter_file: str) -> nn.Module:
+    """
+    Apply LoRA layers to the model.
+
+    Args:
+        model (nn.Module): The neural network model.
+        adapter_file (str): Path to the adapter configuration file.
+
+    Returns:
+        nn.Module: The updated model with LoRA layers applied.
+    """
+    print("load adapters :", adapter_file)
```

Comment:
Nit: is this print here intentionally or debugging? Probably good to avoid printing in the API, we could have a logging in the future if it's useful.

Reply:
@awni This was intentional because I feel load adapter does not have any output to indicate whether it was successful or not. However, I am happy to remove it and use proper logging later. P.S.: I constantly make mistakes by misspelling adapters as "adapter." It's kind of helpful for troubleshooting.

Reply:
I see, is the below `mx.load` error if it can't find the file not useful for that? We could do a check for that instead (e.g. check if the raise error with helpful message if not?

Reply:
> I see, is the below `mx.load` error if it can't find the file not useful for that? We could do a check for that instead (e.g. check if the raise error with helpful message if not?

The mx.load error is not a very user-friendly error message. If no file is found, it just throws `ValueError: [load_npz] Input must be a zip file or a file-like object that can be opened with zipfile.ZipFile`.

Reply:
Oh that's so useless!!! We should fix that indeed

Reply:
@awni I removed the print adapter file path and added a file exist check before mx.load.